packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.test.tsx

```diff
@@ -8,14 +9,15 @@ import { createLocaleContext, LocaleContext, LocaleContextType } from '@bigcomme
 import { CheckoutProvider , PaymentFormValues } from '@bigcommerce/checkout/payment-integration-api';
 
 import { getStoreConfig } from '../../config/config.mock';
-import { CreditCardIconList } from '../creditCard';
 import { getPaymentMethod } from '../payment-methods.mock';
 
 import getPaymentMethodName from './getPaymentMethodName';
 import PaymentMethodId from './PaymentMethodId';
 import PaymentMethodTitle, { PaymentMethodTitleProps } from './PaymentMethodTitle';
 import PaymentMethodType from './PaymentMethodType';
 
+configure({testIdAttribute: 'data-test'})
```

Comment:
@bc-peng @animesh1987 I would recommend do this change on the global level, since we have data-test as test-id in our aplication.

Reply:
Sure, please put it on the global level.
However, it is not a offical recommended approach to use test IDs, we should always aim to use `getByRole` frist.
https://testing-library.com/docs/queries/about/#priority

Reply:
@bc-peng I don't know where I should put it. Could you help me with the configuration?
Whole test is base on the data-test attribute so I decided to just make this refactoring with shortest part.

Reply:
@bc-peng updated with roles